openmp/runtime/src/z_Linux_asm.S

```diff
@@ -1797,13 +1807,13 @@ __kmp_invoke_microtask:
 	.cfi_startproc
 
 	// First, save ra and fp
-	addi	sp, sp, -16
-	sd	ra, 8(sp)
-	sd	fp, 0(sp)
-	addi	fp, sp, 16
+	addi	sp, sp, -2*SZREG
+	REG_S	ra, 1*SZREG(sp)
+	REG_S	fp, 0*SZREG(sp)
+	addi	fp, sp, 2*SZREG
 	.cfi_def_cfa	fp, 0
-	.cfi_offset	ra, -8
-	.cfi_offset	fp, -16
+	.cfi_offset	ra, -1*SZREG
+	.cfi_offset	fp, -2*SZREG
 
 	// Compute the dynamic stack size:
 	//
```

Comment:
Is the "8 bytes" for in the comment `We need 8 bytes for each argument that cannot be passed to the 'pkfn'` below still accurate?